app/javascript/events.js

```diff
@@ -1,4 +1,5 @@
 let cachedEvents = JSON.parse(localStorage.getItem("cachedEvents")) || [];
+console.log("Cached Events from localStorage:", cachedEvents); 
```

Comment:
Is this for debugging purposes?

Reply:
yes i had to include it to check whether the json file is loading user in the console, it wont have any effect on the front end

Reply:
If there are no bugs in the final version of the code, rmeove any debug statements. We don't wanna stuff unrelated stuff in codebase